src/software/ai/hl/stp/play/example_play.cpp

```diff
@@ -5,16 +5,36 @@
 
 const std::string ExamplePlay::name = "Example Play";
 
+/**
+ * This function returns name of the play
+ * @return ExamplePlay's name (which is "Example Play" as seen on Line 6)
+ */
 std::string ExamplePlay::getName() const
 {
     return ExamplePlay::name;
 }
 
+/**
+ * This function returns whether the play is applicable
+ * Applicable describes the conditions for the play should start
+ * For instance, the play should start when it is kick-off
+ * @return If ExamplePlay is applicable
```

Comment:
I think the comment still needs a `@param world` since there is a parameter being passed into the function. Also, this section of the comment should be in `.h` file